capa/features/extractors/dnfile/insn.py

```diff
@@ -139,9 +139,9 @@ def extract_insn_property_features(fh: FunctionHandle, bh, ih: InsnHandle) -> It
             if property is None:
                 return
             if property[1] == DnPropertyAccessType.SET:
-                yield WriteProperty(str(property[0])), ih.address
+                yield Property(str(property[0]), "property/write"), ih.address
             elif property[1] == DnPropertyAccessType.GET:
```

Comment:
what about the "other" access types? Should we still emit them here or only care about read and write?